### PR TITLE
updates static promo message on checkout page

### PR DESCRIPTION
**Pull request by taehyunlim**
Screen Shot 2020-10-08 at 5 25 31 PM

### CODE REVIEW

File: cartridges/int_affirm_controllers/cartridge/templates/default/affirm/paymentmethodinput.isml [hunk 1]
```diff
@@ -29,10 +29,9 @@
 			<img alt="${Resource.msg(paymentMethodType.label,'forms',null)}" 
 				src="https://cdn-assets.affirm.com/images/blue_logo-transparent_bg.png" 
 				title="${Resource.msg(paymentMethodType.label,'forms',null)}"/>
-			<span>${Resource.msg('payment.monthlypayments', 'affirm', null)}</span>
-			<a class="affirm-product-modal" style=""
-			data-amount="${pdict.Basket.totalGrossPrice.multiply(100).getValue().toFixed()}"
-			>${Resource.msg('payment.learnmore', 'affirm', null)}</a>
+			<span>${Resource.msg('payment.payovertime', 'affirm', null)}</span>
+			<a data-page-type="checkout" data-amount="${pdict.Basket.totalGrossPrice.multiply(100).getValue().toFixed()}"
```
Comment (kkong87) [inline cartridges/int_affirm_controllers/cartridge/templates/default/affirm/paymentmethodinput.isml:33]: Do we want to keep the class and just add the data-page-type

File: cartridges/int_affirm_controllers/cartridge/templates/default/affirm/paymentmethodinput.isml [hunk 2]
```diff
@@ -29,10 +29,9 @@
 			<img alt="${Resource.msg(paymentMethodType.label,'forms',null)}" 
 				src="https://cdn-assets.affirm.com/images/blue_logo-transparent_bg.png" 
 				title="${Resource.msg(paymentMethodType.label,'forms',null)}"/>
-			<span>${Resource.msg('payment.monthlypayments', 'affirm', null)}</span>
-			<a class="affirm-product-modal" style=""
-			data-amount="${pdict.Basket.totalGrossPrice.multiply(100).getValue().toFixed()}"
-			>${Resource.msg('payment.learnmore', 'affirm', null)}</a>
```
Comment (kkong87) [inline cartridges/int_affirm_controllers/cartridge/templates/default/affirm/paymentmethodinput.isml:35]: was there a reason why we got rid of that?

File: cartridges/int_affirm_pipelines/cartridge/templates/default/affirm/paymentmethodinput.isml [hunk 3]
```diff
@@ -29,10 +29,9 @@
 			<img alt="${Resource.msg(paymentMethodType.label,'forms',null)}" 
 				src="https://cdn-assets.affirm.com/images/blue_logo-transparent_bg.png" 
 				title="${Resource.msg(paymentMethodType.label,'forms',null)}"/>
-			<span>${Resource.msg('payment.monthlypayments', 'affirm', null)}</span>
-			<a class="affirm-product-modal" style=""
-			data-amount="${pdict.Basket.totalGrossPrice.multiply(100).getValue().toFixed()}"
-			>${Resource.msg('payment.learnmore', 'affirm', null)}</a>
+			<span>${Resource.msg('payment.payovertime', 'affirm', null)}</span>
+			<a data-page-type="checkout" data-amount="${pdict.Basket.totalGrossPrice.multiply(100).getValue().toFixed()}"
```
Comment (kkong87) [inline cartridges/int_affirm_pipelines/cartridge/templates/default/affirm/paymentmethodinput.isml:33]: Same as above

File: cartridges/int_affirm_sfra/cartridge/templates/default/affirm/paymentMethodInputMF.isml [hunk 4]
```diff
@@ -22,9 +22,9 @@
 							alt="${Resource.msg('paymentMethodType.label','forms',null)}"
 						src="https://cdn-assets.affirm.com/images/blue_logo-transparent_bg.png"
 						title="${Resource.msg('paymentMethodType.label','forms',null)}"/>
-					<span>${Resource.msg('payment.monthlypayments', 'affirm', null)}</span>
-					<a class="affirm-product-modal" style="" data-amount="${require('dw/order/BasketMgr').currentBasket.totalGrossPrice.multiply(100).getValue().toFixed()}"
-					>${Resource.msg('payment.learnmore', 'affirm', null)}</a>
+					<span>${Resource.msg('payment.payovertime', 'affirm', null)}</span>
+					<a data-page-type="checkout" data-amount="${require('dw/order/BasketMgr').currentBasket.totalGrossPrice.multiply(100).getValue().toFixed()}"
```
Comment (kkong87) [inline cartridges/int_affirm_sfra/cartridge/templates/default/affirm/paymentMethodInputMF.isml:26]: Same as above

**Review by kkong87 — CHANGES_REQUESTED** [review]

File: cartridges/int_affirm_controllers/cartridge/templates/default/affirm/paymentmethodinput.isml (hunk 1 above)
Comment (taehyunlim) [reply cartridges/int_affirm_controllers/cartridge/templates/default/affirm/paymentmethodinput.isml:33]: Yes, it makes sense to keep the class. I've restored it for all three versions (SFRA, Pipelines, Controllers)

File: cartridges/int_affirm_controllers/cartridge/templates/default/affirm/paymentmethodinput.isml (hunk 2 above)
Comment (taehyunlim) [reply cartridges/int_affirm_controllers/cartridge/templates/default/affirm/paymentmethodinput.isml:35]: Restored the class

**Comment by kkong87** [comment]
LGTM

**Comment by taehyunlim** [comment]
Moved to the new PR to avoid confusion:
https://github.com/Affirm/Salesforce_Commerce_Cloud_Affirm/pull/39

**Pull request closed without merging**